pinot-query-planner/src/main/java/org/apache/pinot/query/catalog/PinotCatalog.java

```diff
@@ -127,4 +128,20 @@ public boolean isMutable() {
   public Schema snapshot(SchemaVersion version) {
     return this;
   }
+
+  private static boolean isNullEnabled(TableCache tableCache, String rawTableName) {
```

Comment:
This function gets a table property. Other similar functions are member functions. Do we want change this function to a member function for consistency?